stl/inc/complex

```diff
@@ -635,46 +635,45 @@ public:
         : _Complex_base<float, _Fcomplex_value>(
             static_cast<float>(_Right._Val[_RE]), static_cast<float>(_Right._Val[_IM])) {}
 
-    complex<_Ty>& operator=(const _Ty& _Right) {
-
+    _CONSTEXPR20 complex<_Ty>& operator=(const _Ty& _Right) {

```

Comment:
Pre-existing: This should use the injected-class-name and return `complex&` to be consistent with the compound assignment operators below. I'll comment the two identical occurrences in the other specializations.